java-maven-sonar-argocd-helm-k8s/spring-boot-app/JenkinsFile

```diff
@@ -21,7 +21,7 @@ pipeline {
     }
     stage('Static Code Analysis') {
       environment {
-        SONAR_URL = "http://34.201.116.83:9000"
```

Comment:
By Mentioning this IP Address (34.201.116.83) will give a user clear view for defining Its a IP Address block to replace their IP bro